app/readme.md

```diff
@@ -37,7 +37,7 @@ Install `docker-compose` from <https://docs.docker.com/compose/install/>.
 If you are on **macoS**, **Linux** or **WSL2**, run the following command:
 
 ```sh
-docker run --rm -w /app -v $(pwd):/app couchers/grpc ./generate_protos.sh
+docker run --rm -w /app -v $(pwd):/app registry.gitlab.com/couchers/grpc ./generate_protos.sh
```

Comment:
Will we need an account with access to our GitLab registry for things to work? Or is it public like the Docker registry we used before?

Reply:
It's a public github registry: so no need to log in!